x-pack/test/functional_with_es_ssl/page_objects/triggers_actions_ui_page.ts

```diff
@@ -157,5 +157,34 @@ export function TriggersActionsPageProvider({ getService }: FtrProviderContext)
       );
       await createBtn.click();
     },
+    async setAlertName(value: string) {
+      await testSubjects.setValue('alertNameInput', value);
+      await this.assertAlertName(value);
+    },
+    async assertAlertName(expectedValue: string) {
+      const actualValue = await testSubjects.getAttribute('alertNameInput', 'value');
+      expect(actualValue).to.eql(expectedValue);
+    },
+    async setAlertInterval(value: number, unit?: 's' | 'm' | 'h' | 'd') {
+      await testSubjects.setValue('intervalInput', value.toString());
+      if (unit) {
+        await testSubjects.selectValue('intervalInputUnit', unit);
+      }
+      await this.assertAlertInterval(value, unit);
+    },
+    async assertAlertInterval(expectedValue: number, expectedUnit?: 's' | 'm' | 'h' | 'd') {
+      const actualValue = await testSubjects.getAttribute('intervalInput', 'value');
+      expect(actualValue).to.eql(expectedValue);
+      if (expectedUnit) {
+        const actualUnitValue = await testSubjects.getAttribute('intervalInputUnit', 'value');
+        expect(actualUnitValue).to.eql(expectedUnit);
+      }
+    },
+    async saveAlert() {
+      await testSubjects.click('saveAlertButton');
+      const isConfirmationModalVisible = await testSubjects.isDisplayed('confirmAlertSaveModal');
+      expect(isConfirmationModalVisible).to.eql(true, 'Expect confirmation modal to be visible');
+      await testSubjects.click('confirmModalConfirmButton');
```

Comment:
Would be good to have two more assertions here after the click. I'd suggest to check that
- the modal is not displayed anymore
- the alert creation flyout is closed (probably needs some waiting - maybe implement in a separate method?)